readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -50,12 +48,6 @@ if 'html_static_path' in globals():
 else:
     html_static_path = ['_static', '{{ static_path }}']
 
-# Add RTD Theme Path. 
-if 'html_theme_path' in globals():
-    html_theme_path.append('{{ template_path }}')
-else:
-    html_theme_path = ['_themes', '{{ template_path }}']
-
```

Comment:
If this is still an issue, open a ticket, it will have more visibility. I believe we deployed a fix for this yesterday.